wordpress-mobile/WordPress-iOS
# Cleaned up i18n text in PersonViewController

Fixes #8165 

<img width="311" alt="screen shot 2017-11-15 at 5 08 33 pm" src="https://user-images.githubusercontent.com/154014/32865299-1f2a44b4-ca28-11e7-9321-205a17811fc7.png"><img width="317" alt="screen shot 2017-11-15 at 5 08 48 pm" src="https://user-images.githubusercontent.com/154014/32865304-2153f1d6-ca28-11e7-8d4b-57417ee86393.png">

Cleans up i18n strings in `PersonViewController`. See the issue ☝️ for more details.

**To test:**

1. Try to remove a given site's user/follower/viewer in the People section of site settings.
2. Review the warning dialog that appears. Make sure everything looks ok.
3. Verify i18n strings output.

Needs review @elibud. Thanks!!!
